google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/AddCustomerMatchUserList.java

```diff
@@ -256,6 +256,8 @@ private void addUsersToCustomerMatchUserList(
       System.out.println("Waiting until operation completes.");
 
       // The polling future implements a default back-off policy for retrying.
+      // Note: Retrieving the results of the asynchronous request is for demonstrative purposes only
+      // as it may take several hours for the user list to be populated with the users.
```

Comment:
nit: replace

"as it may take several hours for the user list to be populated with the users" 

with 

"as the job may not complete for several hours"

This converts to active instead of passive voice, and avoids the common misunderstanding that all users are guaranteed to match and wind up in the list.